mongo/integration/search_index_prose_test.go

```diff
@@ -250,4 +268,47 @@ func TestSearchIndexProse(t *testing.T) {
 		err = collection.SearchIndexes().DropOne(ctx, "foo")
 		require.NoError(mt, err)
 	})
+
+	mt.RunOpts("case 6: Driver can successfully create and list search indexes with non-default readConcern and writeConcern",
+		mtest.NewOptions().CollectionOptions(options.Collection().SetWriteConcern(writeconcern.New(writeconcern.W(1))).SetReadConcern(readconcern.Majority())),
+		func(mt *mtest.T) {
+			ctx := context.Background()
+
+			_, err := mt.Coll.InsertOne(ctx, bson.D{})
+			require.NoError(mt, err, "failed to insert")
+
+			view := mt.Coll.SearchIndexes()
+
+			definition := bson.D{{"mappings", bson.D{{"dynamic", false}}}}
+			searchName := "test-search-index6"
```

Comment:
```suggestion
			const searchName = "test-search-index-case6'"
```

Suggest constantizing and using the name specified in the prose test: "test-search-index-case6'"